.pipelines/e2e.yml

```diff
@@ -86,14 +111,20 @@ jobs:
           run_selenium
           validate_selenium_running
 
+          update_role_sets
+
```

Comment:
Do we need to update the role sets twice? (see lines 119-120 as well)

Reply:
Probably not, good catch :)